index.html

```diff
@@ -4813,13 +4813,45 @@ <h2>Contexts, Types, and Credential Schemas</h2>
       <section class="informative">
         <h3>Base Context</h3>
           <p>
-The base context, located at
-<code>https://www.w3.org/ns/credentials/v2</code> with a SHA-256 digest of
+The base context, located at <code>https://www.w3.org/ns/credentials/v2</code>
+with a SHA-256 digest of
 <strong><code>944167aaabd904ea9e35c98fd7e8794eb6dd42ae4666b036b171e87fc34cc7cc</code></strong>,
 can be used to implement a local cached copy. It is possible to confirm the
 SHA-256 digest by running the following command from a modern Unix command
-interface line: `curl -s https://www.w3.org/ns/credentials/v2 | sha256sum`. For
-convenience, the base context is also provided below.
+interface line: `curl -s https://www.w3.org/ns/credentials/v2 | sha256sum`. It
+is strongly advised that all JSON-LD Contexts used in an application utilize a
+similar mechanism to ensure end-to-end security.
+          </p>
+          <p>
+This section serves as a reminder of the importance of ensuring that, when
+verifying <a>verifiable credentials</a> and <a>verifiable presentations</a>, the
+<a>verifier</a> has information that is consistent with what the <a>issuer</a>
+or <a>holder</a> had when securing the <a>credential</a> or <a>presentation</a>.
+This information might include at least: 1) contents of the credential itself;
+2) any content linked to via a URL in the credential; 3) any content linked to
+via a URL, which then links to more content via a URL (and so on).
+          </p>
+          <p>
+Mechanisms used to secure <a>credentials</a> and <a>presentations</a>, such
+[[VC-JWT]] and [[VC-DATA-INTEGRITY]], address securing the contents of the
+credential itself (item #1 in the previous paragraph). The SHA-256 digest of
+JSON-LD Contexts, as well as locally cached copies, address item #2 in the
```

Comment:
> It would be nice to not have to jump back / refer back to read the intention. 

Yes. I would suggest bumping the 1-3 into a list, with each information item immediately followed by the mechanism used to address it. Item #3 would need some suggestion of how to address it.

Reply:
Fixed in cb1d4558d1b6e2108a238c74530e1147851aba3f.